bin/varnishtest/vtc_varnish.c

```diff
@@ -654,6 +668,7 @@ varnish_stop(struct varnish *v)
 {
 
 	VARNISH_LAUNCH(v);
+	vsl_catchup(v);
 	vtc_log(v->vl, 2, "Stop");
 	(void)varnish_ask_cli(v, "stop", NULL);
 	wait_stopped(v);
```

Comment:
This seems to drastically slow down the test suite, so test cases needing this should rather opt in.